src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -281,6 +281,7 @@ interface ComponentState {
     secondarySlateValuesMap: SlateValueMap
     isTerminal: boolean
     reprompt: boolean
+    entryNode: boolean
```

Comment:
It would be better to the name of property on action model. Like `isTerminal` above, likewise for new `isEntryNode`
There's general practice of boolean variables being named as yes/no questions that can be answered to imply boolean.  `EntryNode` sounds like it could be of type `Node` but we don't have that concept.

Reply:
Updating in new PR